app/views/mailing_list/steps/completed/shared/_book_a_callback.html.erb

```diff
@@ -1,5 +1,4 @@
-<div class="callback-module">
```

Comment:
I think you need to remove the closing </div> as well in line 11 @emmaachesongray ?